.github/workflows/triage.yml

```
@@ -0,0 +1,53 @@
+name: Triage new issues
+
+on:
+  issues:
+    types: [ opened ]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
+  ORGANIZATION: ethereum
+  PROJECT_NUMBER: 43
+  DRY_RUN: true
+
+jobs:
+  triage_issues:
+    runs-on: ubuntu-latest
+    if: join(github.event.issue.labels) == ''
+    steps:
+      - name: Retrieve project data
+        env:
+          ORGANIZATION: ${{ env.ORGANIZATION }}
+          PROJECT_NUMBER: ${{ env.PROJECT_NUMBER }}
+        run: |
+          gh api graphql --raw-field query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org) {
+                project(number: $number) {
+                  columns(first: 10) {
+                    nodes {
+                      id,
+                      name
+                    }
+                  }
+                }
+              }
+            }' --raw-field org=$ORGANIZATION --field number=$PROJECT_NUMBER >> project_data.json
```

This looked hard to decipher at first with the big query in the middle of it. What do you think about formatting it this way?
```suggestion
          gh api graphql \
            --raw-field org=$ORGANIZATION \
            --field number=$PROJECT_NUMBER \
            --raw-field query='
              query($org: String!, $number: Int!) {
                organization(login: $org) {
                  project(number: $number) {
                    columns(first: 10) {
                      nodes {
                        id,
                        name
                      }
                    }
                  }
                }
              }' >> project_data.json
```

Same with the other query below.

---

Definitely, much better! :)